extensions/cactus-plugin-htlc-coordinator/README.md

```diff
@@ -0,0 +1,3 @@
+# `@hyperledger/cactus-plugin-htlc-coordinator`
+
+Allows Cactus nodes to interact beetwen diferent networks
```

Comment:
@jordigiam A minimal code example / reference to the diagrams could be useful here. The concern: people who come to the HTLC coordinator package from npmjs.com will only see this readme. It's also good if you link here to other parts of the docs (the readthedocs site for example) just make sure to point people somewhere for a little extra reading.

Reply:
Done